lib/src/main/java/graphql/nadel/engine/NadelExecutionContext.kt

```diff
@@ -19,9 +20,15 @@ data class NadelExecutionContext internal constructor(
     val hints: NadelExecutionHints,
     val instrumentationState: InstrumentationState?,
     internal val timer: NadelInstrumentationTimer,
+    internal val incrementalResultSupport: NadelIncrementalResultSupport,
+    internal val hydrationDetails: ServiceExecutionHydrationDetails? = null,
 ) {
     private val serviceContexts = ConcurrentHashMap<String, CompletableFuture<Any?>>()
 
+    @Deprecated("Use incrementalSupport instead", ReplaceWith("incrementalResultSupport"))
+    internal val deferSupport: NadelIncrementalResultSupport
```

Comment:
I renamed it, but this is just so other branches can still use the code.

I think Steven's branch relies on it. I'll delete this when we merge those PRs.